Database Scripts/V016__Insert_Data_Action_Type.sql

```diff
@@ -0,0 +1,9 @@
+USE BeantasticDB;
+GO
+
+INSERT INTO [dbo].[ActionType] ([description], [markiplier])
```

Comment:
Not sure how markiplier relates to this lol or what those values relate to 🤔

Reply:
Multiplier we just thought it would be funny and I need it for a tech away.